install_dep_packages.sh

```diff
@@ -134,10 +130,11 @@ echo "####  Cloning, Building and Installing the '$MODULE' module ####"
 mkdir -p "${SRC_DIR}"/"$MODULE"
 git clone https://github.com/google/protobuf.git "$SRC_DIR"/"$MODULE"
 cd "$SRC_DIR"/"$MODULE"
-git checkout tags/v3.6.1
-./autogen.sh
-./configure "$CONFIG_PREFIX"
-make "$NUM_THREADS"
+git checkout tags/v3.18.1
```

Comment:
The following command would reduce the amount of data (and time) it takes to download the `v3.18.1` branch.
```bash
git clone --depth=1 -b v3.18.1 https://github.com/google/protobuf.git "$SRC_DIR"/"$MODULE"
```

Reply:
@rst0git I wasn't aware of depth parameter. Good to know, thanks for sharing!! Will try to use it wherever possible!